documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,287 @@
+
+
+.Migrating VM from one OCP-V cluster to another using timeouts and fails
+
+When trying to migrate VMs from one OCP-V cluster to another using {product-short}, the migration could timeout and fail. link:https://issues.redhat.com/browse/MTV-1052[(MTV-1052)]
+
+
+.Guest operating system from vSphere might be missing
+
+The guest operating system may be missing for VMs from vSphere. vSphere stops reporting the operating system after not receiving updates from VMware tools after some time. link:https://issues.redhat.com/browse/MTV-1046[(MTV-1046)]
+
+
+.{product-short} forklift-controller pod crashes in a loop after configuring vSphere provider
+
+The {product-short} `forklift-controller` pod crashes in a loop, and the product is unusable if vSphere provides disk with null datastore. link:https://issues.redhat.com/browse/MTV-1029[(MTV-1029)]
+
+
+
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12430275[Known Issues] in Jira.
+
+
+
+[id="resolved-issues-26_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+
+.Deleting a provider automatically also deletes the PV
+
+In the previous versions of {product-short}, the PV was not removed when the OVA provider was deleted. This has been resolved in {product-short} 2.6.0, and the PV is automatically deleted when the OVA provider is deleted. link:https://issues.redhat.com/browse/MTV-848[(MTV-848)]
+
+
+.Data loss is no longer possible when migrating VMware VMs with snapshots
+
+In the previous versions of {product-short}, when migrating a VMware VM that had a snapshot, the VM that was created in {ocp-name} Virtualization contained the data in the snapshot, but not in the running VM that was migrated. This has been resolved in {product-short} 2.6.0.  link:https://issues.redhat.com/browse/MTV-447[(MTV-447)]
+
+
+.Adding an OVA provider no longer displays the `ConnectionTestFailed` error message
+
+In the previous versions of {product-short}, when adding an OVA provider, `The provider is not ready - ConnectionTestFailed` error message appeared, even though the provider was eventually created. The message disappeared after a few seconds. This has been resolved in {product-short} 2.6.0. link:https://issues.redhat.com/browse/MTV-671[(MTV-671)]
+
+
+.Canceling and deleting a failed migration plan does not clean up the `populate` pods and PVC
+
+In earlier releases of {project-short}, when you canceled and deleted a failed migration plan, and after creating a PVC and spawning the `populate` pods, the `populate` pods and PVC were not deleted. You had to delete the pods and PVC manually. This issue has been resolved in {project-short} 2.6.0. link:https://issues.redhat.com/browse/MTV-678[(MTV-678)]
+
+
+.{ocp} to {ocp} migrations require the cluster version to be 4.13 or later
```

Comment:
this item already appear above as `.Migrations from {ocp-name} versions before 4.13 are now blocked`, I think it's better be documented as a resolved issue than a feature, but in both cases it should say something about a validation that is now shown when trying to migration from {ocp-nane} < 4.13